white_label/brands/mabulle/js/locales/fr.json

```diff
@@ -0,0 +1,9 @@
+{
+  "screens": {
+    "welcome": {
+      "body": "Au sein de votre espace Ma Bulle, vous êtes chez vous!<br /><br />Rangez vos papiers, gérez votre consommation énergétique, suivez vos dépenses… La Métropole de Lyon vous propose un service pour vous faciliter votre vie numérique.",
```

Comment:
J'imagine qu'il faut un espace entre vous et le "!" (insécable?)

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/489df6ad7083f78a6dff324b1780b642819a262e..f9e185be78c78b3a9d7ff64f6a360b4ac56cca92